### DISCUSSION TITLE
Itables returns class in dashboard

### DISCUSSION THREAD

**Discussion by XanderC137** [quarto-dev/quarto-cli#7723] [Q&A] [answered]
### Description

I have some data in a dataframe that I want to display in my dashboard using itables. When following the documentation, the only output I get is the class. I thought it might have been too big so I made a dummy dataframe like below but this gives the exact same problem.

````{qmd}
---
title: "<b>Classified cryostock isolates</b>"
format: dashboard
scrolling: true
---
```{python}
import pandas as pd
from itables import show

table_df = pd.DataFrame(data = {'Col1':[1,2,3,4],'Col2':[1,2,3,4]})
show(table_df)
```
````

Output:
<img width="949" alt="Scherm­afbeelding 2023-11-27 om 14 36 26" src="https://github.com/quarto-dev/quarto-cli/assets/151828460/918fb049-005c-423a-ae02-5cc56ed2ebb3">

<img width="1470" alt="Scherm­afbeelding 2023-11-27 om 14 44 04" src="https://github.com/quarto-dev/quarto-cli/assets/151828460/fdb488c7-9304-49a5-b828-63016ce3aaf8">

**Comment by mcanouil** [comment, marked as answer]
I cannot reproduce with your example which does not match your screenshots.
you are likely using an old version of something.
Note that you should not use raw HTML outside of raw code blocks or raw inline element.
Use markdown syntax to get your title in bold.

<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/8c19d5e9-89b9-4a1f-905b-5cf8c15e5957">
